JoyboyCommunity/src/assets/icons.tsx

```diff
@@ -189,3 +189,64 @@ export const SendIcon: React.FC<SvgProps> = (props) => {
     </Svg>
   );
 };
+
+export const RepostIcon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg width="32" height="32" viewBox="0 0 21 21" {...props}>
+      <G
+        fill="none"
+        fill-rule="evenodd"
+        stroke="currentColor"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      >
+        <Path d="m13.5 13.5l3 3l3-3" />
+        <Path d="M9.5 4.5h3a4 4 0 0 1 4 4v8m-9-9l-3-3l-3 3" />
+        <Path d="M11.5 16.5h-3a4 4 0 0 1-4-4v-8" />
+      </G>
+    </Svg>
+  );
+};
+
+export const LikeIcon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg width="26" height="26" viewBox="0 0 20 20" fill="none" {...props}>
+      <Path
+        d="M15.7141 3.12058C12.3877 1.33653 9.99929 3.96642 9.99929 3.96642C9.99929 3.96642 7.61073 1.33652 4.28431 3.12057C0.255387 5.2814 -0.0339221 13.7433 9.99929 17.5C20.0325 13.7433 19.743 5.28141 15.7141 3.12058Z"
+        stroke="#14142C"
```

Comment:
colors should be currentColor, and the usage should include the color, not the definition. For example, this should be `stroke="currentColor"` and usage would be `<LikeIcon color="#14142C" />`